### DISCUSSION TITLE
"Your browser is out of date!" error

### DISCUSSION THREAD

**Discussion by wss-test** [help-me-mom/ng-mocks#9130] [Q&A]
I am having issues accessing https://ng-mocks.sudo.eu/ if going via proxy cloud (Cloud SWG, via one of these addresses: https://knowledge.broadcom.com/external/article/167174). The error is present on chrome/edge browsers, doesn't appear on Firefox. It doesn't change if I update my browser to the latest release (125.0.6422.142).

This is the exact error I see on latest chrome (left) vs what it shows on older firefox (right):
![image](https://github.com/help-me-mom/ng-mocks/assets/44369833/b3ea4aff-233d-4ba6-a02e-c4d2d20657df)

Is this related to Cloudflare policy (blocking chrome access from certain IP range) which should be raised with Cloudflare?

**Comment by satanTime** [comment]
Hi. Thanks for reporting. I guess it's related to docosaurus which I use to build documentation. 

I'll take a look what I can improve.

**Reply by wss-test** [reply]
hi, did you manage to look at this?

**Reply by satanTime** [reply]
Hi, I've tried in the latest safari, chrome and firefox - all work well.

**Reply by sanymil1969** [reply]
It is not working in my enviroment. Please check https://github.com/help-me-mom/ng-mocks/discussions/9444
